transcoder/basisu_transcoder.cpp

```diff
@@ -7387,7 +7387,7 @@ namespace basist
 
 					uint32_t m = (le * 5 + he * 3) / 8;
 
-					int err = labs((int)v - (int)m);
```

Comment:
Is there a reason `abs()` isn't used here instead of the `long` version?

Reply:
Yes - I've had nothing but problems over the years using abs() on various platforms/compilers. Traumatized!